libs/workordersidesheet/src/lib/ui-sidesheet/WorkorderSidesheet.tsx

```diff
@@ -73,6 +87,28 @@ export const WorkorderSidesheet = (props: {
     }
   );
 
+  const mccrIcon = (status: string) => {
+    return <h3>{status}</h3>;
+  };
+
+  const tagsOverlay = useMemo(() => {
+    return modelConfig?.tags?.map((tag) => ({
+      tagNo: tag.tagNo,
+      description: tag.description,
+      status: tag.status,
+      icon: mccrIcon(tag.status || ''),
+    })) as TagOverlay[];
+  }, [mccr]);

```

Comment:
Just personal preference I guess but is it really necessary memoizing this? Doesnt look like a "heavy computation"

Reply:
Removed memorization